docs/dataset-overviews/amip-sst-sea-ice-boundary-forcing.md

```diff
@@ -3,64 +3,71 @@
 <!--- source_id_stub="PCMDI-AMIP" -->
 # AMIP sea-surface temperature and sea-ice boundary forcing
 
-**This section is a work in progress.**
-**For a first draft, see https://github.com/PCMDI/input4MIPs_CVs/pull/146**
-
 ## Key contacts
 
-- Names: Paul Durack
+- Names: Paul J. Durack
 - Emails: durack1@llnl.gov
 
 ## Summary
 
 <!--- begin-cmip7-phases-source-ids -->
 <!--- Do not edit this section, it is automatically updated when the docs are built -->
-### Source IDs for CMIP7 phases
+### Source IDs for CMIP phases
```

Comment:
I would leave this as CMIP7, as we're not going to start providing information about CMIP5 or CMIP6 here (at least not in the short term)

Reply:
We already have CMIP7 and CMIP6Plus as headings on the page further down.. So happy to iterate